python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -181,6 +182,43 @@ class PipesException(TypedDict):
 # ##### UTIL
 # ########################
 
+ESCAPE_CHARACTER = "\\"
+
+
+def de_escape_asset_key(asset_key: str) -> str:
+    r"""Removes the backward slashes escape characters from the asset key.
+
+    Example: "foo\\/bar" -> "foo/bar"
+    """
+    return asset_key.replace(ESCAPE_CHARACTER, "")
+
+
+def to_assey_key_path(asset_key: str) -> List[str]:
+    """Converts an asset key to a collection of key parts.
+
+    Forward slash (except escaped) is used as separator. De-escapes the key.
```

Comment:
Seems odd that this "de-escapes" the key but doesn't share logic with `de_escape_asset_key`

Reply:
I don't see a way to reuse the other de-escaping code here, because splitting and escaping can't really be done separately.